bin/conjur-cli.rb

```diff
@@ -230,14 +231,24 @@
 
 desc 'Manage Conjur configuration'
 command :configuration do |cgrp|
+  Anyway::Settings.default_config_path = "/etc/conjur/config"
+
+  begin
+    conjur_config = Conjur::ConjurConfig.new
+  rescue Conjur::ConfigValidationError => e
+    $stderr.puts e
+    exit 1
+  end
+
   cgrp.desc 'Show Conjur configuration attributes and their sources'
   cgrp.long_desc(<<~DESC)
-    Show Conjur configuration attributes and their sources.
+    Validate Conjur configuration then show configuration attributes and their
+    sources.
 
     The values displayed by this command reflect the current state of the
-    configuration sources. For the example, the environment variables and
-    config file. These may not reflect the current values used by the running
-    Conjur server.
+    configuration sources. For example, the environment variables and config
+    file. These may not reflect the current values used by the running Conjur
+    server.
```

Comment:
Is there a way to inspect these?  If so, we should explain how or link to relevant docs.

Reply:
Not currently, no, though I have proposed adding an endpoint to the API for doing so. I believe it will be future work.